runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -668,46 +668,65 @@ fn action_function_call_base(ctx: &mut EstimatorContext) -> GasCost {
     total_cost.saturating_sub(&base_cost, &NonNegativeTolerance::PER_MILLE)
 }
 fn action_function_call_per_byte(ctx: &mut EstimatorContext) -> GasCost {
-    let total_cost = {
-        let mut make_transaction = |tb: &mut TransactionBuilder| -> SignedTransaction {
-            let sender = tb.random_unused_account();
-            tb.transaction_from_function_call(sender, "noop", vec![0; 1024 * 1024])
-        };
-        transaction_cost(ctx, &mut make_transaction)
-    };
-
-    let base_cost = noop_function_call_cost(ctx);
+    let xs = [1, 1_000_000, 4_000_000];
```

Comment:
Hm, naively, just 3 points seems a bit too little for least squares. Do we want more? Or do we want just two points and simple division? 

Actually, that's intereresting -- a lot of per-byte costs are computed as `(10k - base) / 10k`, but that's just a degenerate case of least squares!

Reply:
Agreed that it is a bit odd, so I added a comment to explain the motivation.

The problem is that adding more X values below 1M increases overall variance because often f(x=500k) will be smaller than f(x=100k) in time-based estimation. So I picked three values that are at least sorted.

Of course, making the estimation more stable would be ideal. But since it is so insignificantly small, I couldn't really justify spending the time on this specific estimate.